Yank/experiment.py

```diff
@@ -5,8 +5,17 @@
 # =============================================================================
 
 """
+Experiment
+==========
+
 Tools to build Yank experiments from a YAML configuration file.
 
+This is not something that should be normally invoked by the user, and instead
+created by going through the Command Line Interface with the ``yank script``.
+
+These functions are not designed to be in fully featured API, but we document
```

Comment:
We document all functions, even private ones, so that developers know what the functions do. I don't think it's necessary to say "we are documenting these functions".